modules/virt-example-inherit-static-ip-from-nic.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * virt/node_network/virt-updating-node-network-config.adoc
+
+:_content-type: REFERENCE
+[id="virt-example-inherit-static-ip-from-nic_{context}"]
+= Example: Linux bridge interface node network configuration policy to inherit static IP address from the NIC attached to the bridge
```

Comment:
I imagine this would be hard to shorten, but I think the current length makes it less readable. Maybe something like "Example: Linux bridge configured to inherit its NIC's static IP address"?

Reply:
Yeah, that's part of the reason I kept the long title, to keep things consistent.